packages/core/src/browser/status-bar/status-bar.tsx

```diff
@@ -195,7 +195,7 @@ export class StatusBarImpl extends ReactWidget implements StatusBar {
             if (!(typeof val === 'string') && LabelIcon.is(val)) {
                 const octicon = getIconByName(val.name);
                 if (octicon) {
-                    children.push(<span key={key} className={val.animation ? 'fa-' + val.animation : ''}><Octicon icon={octicon} /></span>);
+                    children.push(<span key={key} className={val.animation ? 'fa-' + val.animation : 'fa'}><Octicon icon={octicon} height={12.5} width={12.5} /></span>);
```

Comment:
The `fa` class here is to have the same `line-height` as the fontawesome icons. As it's already using `fa-...` for the animation, I thought it would make sense to use `fa` here too.

`12.5` is the size my browser tells me the fontawesome icons are, not sure if there would be a better way to size those SVG icons?

Reply:
I'll ping @kittaakos since he was the original author :)
Thank you for updating the commit message!

Reply:
> he was the original author

I am confused: https://github.com/theia-ide/theia/commit/d4222fd086257306cd2c69bd6ea6c80a71772678#diff-f9ebf8fac439ed099cd352d59b6f2eda

Reply:
> > he was the original author
> 
> I am confused: [d4222fd#diff-f9ebf8fac439ed099cd352d59b6f2eda](https://github.com/theia-ide/theia/commit/d4222fd086257306cd2c69bd6ea6c80a71772678#diff-f9ebf8fac439ed099cd352d59b6f2eda)

Ohh sorry about that, I must have gotten confused.

Reply:
So maybe the original author is @akosyakov ?